src/kkt.c

```diff
@@ -112,7 +112,7 @@ idxint kkt_solve(kkt* KKT, spmat* A, spmat* G, pfloat* Pb, pfloat* dx, pfloat* d
     pfloat ney = 0;
     pfloat nez = 0;
     pfloat nerr;
-    pfloat nerr_prev;
+    pfloat nerr_prev = 0.0f;

```

Comment:
Sorry to comment so late. I've manually merged your PR into the develop branch, but with one change: I changed this line to initialize to `nerr_prev = NAN`. (It had originally been set to NaN by @adomahidi, but we lost it in a subsequent merge.)